tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Secure Connections to Serverless Tier Clusters
+summary: Introduce TLS connection in TiDB Serverless Tier clusters.
+---
+
+# Secure Connections to Serverless Tier Clusters
+
+This document introduces the core information about TLS in TiDB Serverless Tier.
+
+## Can I disable TLS in TiDB Serverless Tier?
+
+No.
+
+TiDB Serverless Tier allows only TLS connections and rejects non-TLS connections. The reason is that users connect to TiDB Serverless Tier clusters through a public network, so it is really important to use TLS to ensure communication security.
+
+## What TLS versions can I use?
+
+TiDB Serverless Tier only supports TLS 1.2 and TLS 1.3. 
+
+## What certificates do I need？
+
+TiDB Serverless Tier uses certificates from [Let's Encrypt](https://letsencrypt.org/) as a Certificate Authority (CA) for TLS connection between clients and TiDB Serverless Tier clusters. Usually, the root certificate ([ISRG Root X1](https://letsencrypt.org/certs/isrgrootx1.pem.txt)) of Let's Encrypt is present in your system's root CA stores. If the client uses the system's root CA stores by default, such as Java and Go, you can easily connect securely to TiDB Serverless Tier clusters without specifying the path of CA roots.
+
+However, some drivers and ORMs do not use the system root CA stores. In those cases, you need to configure the CA root path of the drivers or ORMs to your system root CA stores. For example, when you use [mysqlclient](https://github.com/PyMySQL/mysqlclient) to connect a TiDB Serverless Tier cluster in Python on macOS, you need to set `ca: /etc/ssl/cert.pem` in the `ssl` argument.
+
+> **Note:**
+> 
+> TiDB Serverless Tier does not provide a CA root certificate download, because we don't guarantee that the same CA will be used to issue a certificate in the future, which will cause the CA root certificate to change. 
+>
+> However, TiDB Serverless Tier promises always using a CA root certificate that is commonly available, which is provided in all common systems. 
```

Comment:
Be cautious about using "promises" in the user docs because we could not promise the future. Is there any way to avoid this word in this sentece?

Reply:
I think change to ensure.